### PR TITLE
File path info added for customization

### LINKED ISSUE #8887: Information about file location is missing
# General issue

Feedback on page:[ /guides/v2.4/frontend-dev-guide/layouts/layout-types.html](https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/layouts/layout-types.html)

## Description:

Information missing. Where is the file for the following code at "Page layouts declaration"?
It just shows the code but does not explain where the file is located.

```
<?xml version="1.0"?>
<!--
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
-->
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="3columns" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <update handle="customer_account"/>
    <body>
        <referenceContainer name="content">
            <block class="Magento\Wishlist\Block\Customer\Sharing" name="wishlist.sharing" template="Magento_Wishlist::sharing.phtml" cacheable="false"/>
        </referenceContainer>
    </body>
</page>
```

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) updates the file path for customization of the page layout file in the custom theme or custom module.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/layouts/layout-types.html

Fixes #8887

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Nice

File: src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md [hunk 1]
```diff
@@ -91,6 +91,11 @@ Sample page layout declaration file: `<Magento_Theme_module_dir>/view/frontend/l
 
 Use the `layout` attribute in the `page` node of a page configuration file to define a layout type for the page. The following example shows how to use the `3 columns` page layout type for the [Wish List Sharing]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Wishlist/view/frontend/layout/wishlist_index_share.xml#L8) page:
 
+Override the default `wishlist_index_share.xml` in any one of the following paths and add the `layout="3columns"` in the `page` node.
+
+*  Override the layout in custom theme: `<custom_theme_dir>/Magento_Wishlist/layout/wishlist_index_share.xml`
```
Comment (atwixfirster) [inline src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:96]: Hi, @ajithkumar-maragathavel !

Good PR. However, I have a question from the reader's side. Could you please add a small clarification in which cases I should use `module override` option and in which ones I should use `theme` override?

Thank you!

Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:96]: @atwixfirster Thanks for the suggestions. I have updated the PR with the explanation. Can you please review it and share your suggestion?

Comment (atwixfirster) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md:96]: Hi, @ajithkumar-maragathavel !

Thank you for the adding explanations. I would like to move them into each point.

Please check my vision and let me know if it makes sense :

```
*  Override the layout in custom theme (_in case when a custom-built theme applied on the storefront_): `<custom_theme_dir>/Magento_Wishlist/layout/wishlist_index_share.xml`
 *  Override the layout in custom module (_in case of building third-party extensions and need to make changes to the existing layout_): `<custom_module_dir>/view/frontend/layout/wishlist_index_share.xml`
```

Thank you!

Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md]: Hi, @atwixfirster Your version looks pretty good. I have updated the PR. Thanks

**Review by atwixfirster — APPROVED** [review]

File: src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md]: Thank you, @ajithkumar-maragathavel!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8888`.

**Pull request merged**